handler.go

```diff
@@ -1908,10 +1907,13 @@ func (h *Handler) InputJSONDataParser(req map[string]interface{}, index *Index,
 	for _, field := range inputDef.Fields() {
 		validFields[field.Name] = true
 		if field.PrimaryKey {
-			columnLabel := field.Name
-			value, ok := req[columnLabel]
+			primaryKey := field.Name
```

Comment:
This logic is not correct. We don't want to require the user to provide a `columnID` field in the input definition. The input definition can have any field marked as its `primaryKey`. No matter what that field is called, its value gets used as the column value.